packages/embed/src/validation.test.ts

```diff
@@ -609,15 +609,15 @@ describe('validateIntent()', () => {
   })
 
   test('should return true if the intent is valid', () => {
-    ;['authorize', 'capture'].forEach((value) => {
+    ;['authorize', 'preferAuthorize', 'capture'].forEach((value) => {
       const options = {
         ...defaultOptions,
         value,
         callback: jest.fn(),
       }
       const valid = validateIntent({
         ...defaultOptions,
-        value: 'authorize',
+        value,
```

Comment:
This hardcoded `'authorize'` seems to be a mistake. Is that right? @luca-gr4vy

Reply:
yeah, probably. I see it should use values from the forEach array